__tests__/extension-requests/extension-requests.test.js

```diff
@@ -594,4 +594,25 @@ describe.skip('Tests the new Extension Requests Screen', () => {
 
     expect(originalReasonValue).toBe('b');
   });
+
+  test('Checks whether tooltip is visible on hover', async () => {
```

Comment:
- All the tooltip's use the same css class, so the functionality will remain same.
- Also there a 400ms delay for the tooltip to appear and since there are 4 tooltip it would add another 1.6s delay to the test suite.